### PR TITLE
TODOs done!

### PR DESCRIPTION
**Pull request by stephencelis**
Bumps things after some TODOs were merged.

### CODE REVIEW

File: Sources/PointFree/Admin/Index.swift [hunk 1]
```diff
@@ -46,7 +46,7 @@ let showNewEpisodeEmailMiddleware =
 private let showNewEpisodeView = View<Database.User> { currentUser in
   ul(
     AppEnvironment.current.episodes()
-      .sorted(by: ^\.sequence)
+      .sorted(by: their(^\.sequence))
```
Comment (stephencelis) [inline Sources/PointFree/Admin/Index.swift]: :)

File: Sources/PointFree/Emails/SendEmail.swift [hunk 2]
```diff
@@ -13,7 +13,7 @@ func sendEmail(
   -> EitherIO<Error, Mailgun.SendEmailResponse> {
 
     let (plain, html): (String, String?) =
-      destructure(
+      either3(
```
Comment (stephencelis) [inline Sources/PointFree/Emails/SendEmail.swift]: Not sure if we wanna further generalize these in the future. But it seemed appropriate.

File: Sources/PointFree/Episode/Show.swift [hunk 3]
```diff
@@ -86,7 +86,7 @@ private let videoView = View<(Episode, isEpisodeViewable: Bool)> { episode, isEp
     [
       `class`([Class.size.width100pct]),
       controls(true),
-      playsInline(true),
+      playsinline(true),
```
Comment (stephencelis) [inline Sources/PointFree/Episode/Show.swift]: Just being consistent for now.

File: Sources/PointFree/PageLayout.swift [hunk 4]
```diff
@@ -182,8 +182,8 @@ private func flashClass(for priority: Flash.Priority) -> CssSelector {
 
 private let favicons: [ChildOf<Element.Head>] = [
   link([rel(.value("apple-touch-icon")), sizes("180x180"), href("https://d3rccdn33rt8ze.cloudfront.net/favicons/apple-touch-icon.png")]),
-  link([rel(.value("icon")), type("image/png"), sizes("32x32"), href("https://d3rccdn33rt8ze.cloudfront.net/favicons/favicon-32x32.png")]),
-  link([rel(.value("icon")), type("image/png"), sizes("16x16"), href("https://d3rccdn33rt8ze.cloudfront.net/favicons/favicon-16x16.png")]),
+  link([rel(.value("icon")), type(.png), sizes("32x32"), href("https://d3rccdn33rt8ze.cloudfront.net/favicons/favicon-32x32.png")]),
+  link([rel(.value("icon")), type(.png), sizes("16x16"), href("https://d3rccdn33rt8ze.cloudfront.net/favicons/favicon-16x16.png")]),
```
Comment (stephencelis) [inline Sources/PointFree/PageLayout.swift]: We already had these :)

File: Sources/PointFree/Pricing.swift [hunk 5]
```diff
@@ -396,7 +396,7 @@ private let teamPricingRowView = View<Pricing> { pricing -> Node in
           min(Pricing.validTeamQuantities.lowerBound),
           name("pricing[team]"),
           onchange(
-            """
+            unsafeJavascript: """
```
Comment (stephencelis) [inline Sources/PointFree/Pricing.swift]: Consistent with your other helpers.

File: Sources/PointFree/Subscribe.swift [hunk 6]
```diff
@@ -27,9 +27,10 @@ let subscribeMiddleware =
 private func subscribe(_ conn: Conn<StatusLineOpen, Tuple2<SubscribeData, Database.User>>)
   -> IO<Conn<ResponseEnded, Data>> {
 
-    let (subscribeData, user) = conn.data |> lower
+    let (subscribeData, user) = conn.data
+      |> lower
 
-    return pure(subscribeData)
```
Comment (stephencelis) [inline Sources/PointFree/Subscribe.swift]: For whatever reason this became too complex to compile.

File: Sources/PointFree/TODO.swift [hunk 7]
```diff
@@ -18,71 +18,12 @@ public func array<A>(_ tuple: (A, A, A, A, A, A, A, A, A)) -> [A] {
...
-// todo: HasPlaysInline
-public func playsInline<T>(_ value: Bool) -> Attribute<T> {
-  return .init("playsinline", value)
-}
-public func poster<T>(_ value: String) -> Attribute<T> {
-  return .init("poster", value)
-}
-
 extension FunctionM {
   public static func <¢> <N>(f: @escaping (M) -> N, c: FunctionM) -> FunctionM<A, N> {
```
Comment (stephencelis) [inline Sources/PointFree/TODO.swift]: There are still a bunch of TODOs in here that I was less sure of.

In this particular case `FunctionM` has another `<¢>` defined already, so the overload feels strange.

File: Sources/PointFree/TODO.swift [hunk 8]
```diff
@@ -101,77 +42,15 @@ public func requireSome<A>(
...
-  return f >>> flatMap(g)
-}
-
-extension EitherIO {
-  public func bimap<F, B>(_ f: @escaping (E) -> F, _ g: @escaping (A) -> B) -> EitherIO<F, B> {
-    return .init(run: self.run.map { $0.bimap(f, g) })
-  }
-}
-
 // TODO: Move to PreludeFoundation?
```
Comment (stephencelis) [inline Sources/PointFree/TODO.swift]: TODO:

- [ ] Make `PreludeFoundation`

File: Sources/PointFree/Pricing.swift (hunk 5 above)
Comment (mbrandonw) [reply Sources/PointFree/Pricing.swift]: noice

**Pull request merged**